.pre-commit-config.yaml

```diff
@@ -17,6 +17,7 @@ repos:
           - "--target-version=py39"
           - "--target-version=py310"
 
+
```

Comment:
@Andrew-ang9, you can remove this blank space between the hooks, normally, this wouldn't have any spaces between them.

Reply:
I have went in and gotten rid of this extra space, thank you for letting me know about this!